README.md

```diff
@@ -5,6 +5,19 @@
 
 Go client and cli for [outline](https://www.getoutline.com/).
 
+# Installation
+
+## Installing the client:
+```shell
+$ go get github.com/ioki-mobility/go-outline
+```
+
+## Installing the CLI:
+```shell
+go install github.com/ioki-mobility/go-outline/cli@latest
+```
+NOTE: Very soon CLI binaries would be available for direct download as part of releases.
```

Comment:
Personally I wouldn't do any promises 😁 but fine for me ✅

Reply:
Its okay to be optimistic... :wink: